src/profiles.js

```diff
@@ -26,24 +28,20 @@ async function httpGetAsync(url) {
 
 function pkDataToProfile(data) {
   let profile = {
-    name: data.member.name,
-    color: '#' + data.member.color,
-    tag: data.system.tag,
+    name: data.member.display_name || data.member.name,
+    color: '#' + (data.member.color || 'ffffff'),
+    tag: data.system.tag || '',
     id: data.member.id,
     system: data.system.id,
     status: ProfileStatus.Done,
-    system_color: '#' + data.system.color,
+    system_color: '#' + (data.system.color || 'ffffff'),
     sender: data.sender,
+    raw: data,
```

Comment:
`raw` seems like code smell. It makes it unclear which parts of the PK API we are actually using.